src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import styles from './style.scss';
+
+const baseClass = 'btn-group';
+
+class ButtonGroup extends React.Component {
+  state = {
+    currentIndex: this.props.currentIndex || -1
```

Comment:
please, move this into `const defaultIndex= -1`

Reply:
done 👍